core/src/nodes/collection.rs

```diff
@@ -439,6 +439,17 @@ impl<TInEvent, TOutEvent, THandler> CollectionStream<TInEvent, TOutEvent, THandl
     }
 }
 
+/// Errors for interrupt(). 
```

Comment:
Maybe flesh this out a bit more? 
```suggestion
/// Reach attempt interrupt errors. 
```